package.json

```diff
@@ -5,7 +5,7 @@
     "backbone": "~1.3.2",
     "backbone-validation": "~0.11.5",
     "coffee-script": "1.6.1",
-    "edx-pattern-library": "0.16.1",
+    "edx-pattern-library": "0.16.3",
```

Comment:
Reviewers: @bjacobel and I spoke about which version to include, and we decided upon 0.16.3. He will bring in 0.16.4 with his work to add a Pattern Library search box to the discussion board.